Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,501 @@
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+Facculty support will be necessary. The SPEX propulsion team has consulted faculty about structure and flow system concerns and made the suggested changes. However, the propulsion system
+has not been tested. It is likely that faculty assistance will be necessary to get the propulsion to a functional stage. This would most likely inlcude the design of an optimally expanding
+nozzle. Faculty support was needed during the endevour to fabricate the structure and design the flow system.
+This included the flow system and how it operates. Dr.\ Garrick, MET department head, assisted the team in the design of the flow system and pointed out unnecessary components, such
+as the flow straightener. He said that the flow was at such a high reynolds number that straightening it in the size constraints is impossible with our budget.
+There may be some assistance required if the nozzle performance is far below nominal, but guessing from past work it will not be a problem. This would concern the nozzle design.
+Nozzle design is very difficult for the necessary size for the CGT, about 1 inch in length and 0.5 inches in diameter. Boundary layer effects are dominent. To accurately design them
+on a computer would include high level numerical smulation which beyond the team's skill set. If there are recurring problems here, Dr.\ Olles can be consulted as he has experience with this.
+There may also be the need for assistance during the data analysis phase or if the team chooses to perform flow simulations. But it will be a small amount of help.
+
+If assistance is needed for the data analysis, Dr.\ Olles would be a good professor to ask. He has a PhD in rocket science
+so he could certainly help.
+\subsection{Long-Term Vision}
+\label{sec:vision}
+
+The long term vision of this project is to be a building block to the next phase of space propulsion research. CREST, a mission that was proposed in the
+spring of 2016 by James Parkus, relies on a space propulsion system. It is the reason the SPEX propulsion subsystem was formed. Thus, it is natural for the
+CGT to assist in the furthering of this project. The main conern for CREST was the propulsion necessary to de-orbit quickly enough that SPEX can determine its entry location and therefore estimate its landing location.
+
+This project will also be great for
+the members to put on their resume. At the end of this project a detailed document will be made which outlines the most important aspects of the project.
+Then the RCS will build off the momentum created by this project. And the understanding of space propulsion inside SPEX will continue to grow and flourish.
+
+Once the CGT is complete, the team can move onto the next phase of space propulsion research. This is the RCS\@. Reaction control systems are used very
```

Comment:
This whole section needs to be significantly shortened. RCS deserves its own PDD and should only be directly mentioned in this document, not thoroughly explained.